libs/designer-ui/src/lib/unitTesting/outputMocks/outputMocks.tsx

```diff
@@ -59,9 +48,35 @@ export interface OutputsField extends SettingProps {
   onValueChange?: ChangeHandler;
 }
 
-export const OutputMocks = ({ isMockSupported, nodeId, onActionResultUpdate, outputs, mocks }: OutputMocksProps) => {
-  const intl = useIntl();
+export interface OutputMocksProps {
+  isMockSupported: boolean;
+  nodeId: string;
+  onActionResultUpdate: ActionResultUpdateHandler;
+  outputs: OutputsField[];
+  mocks: OutputMock;
+  errorMessage: string;
+  onErrorMessageChange: (value: string) => void;
```

Comment:
I think you don't need the `onErrorCodeChange` and `onErrorMessageChange` anymore